modules/custom/az_dam/config/quickstart/core.entity_view_display.node.ib_demo_page.teaser.yml

```diff
@@ -0,0 +1,32 @@
+uuid: 37e02b5e-33a3-48e6-986c-d798602aff9a
```

Comment:
```suggestion
```

Reply:
Resolved locally. Will commit for review when all changes have been made.